src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/AbstractSyntaxFactsService.cs

```diff
@@ -162,10 +162,10 @@ public bool IsOnSingleLine(SyntaxNode node, bool fullSpan)
         {
             while (stack.Count > 0)
             {
-                var current = stack.Pop();
-                var currentNodeOrToken = current.nodeOrToken;
-                var currentLeading = current.leading;
-                var currentTrailing = current.trailing;
+                var (nodeOrToken, leading, trailing) = stack.Pop();
```

Comment:
i hate to ask for manual intervention but could we simplify this by deleting the 3 lines below and fixing up the names?

Reply:
indeed!